boards/arm/atsamr21_xpro/atsamr21_xpro.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2019 Benjamin Valentin
+# SPDX-License-Identifier: Apache-2.0
+identifier: atsamr21_xpro
+name: SAM R21 Xplained Pro
+type: mcu
+arch: arm
+ram: 32
+flash: 256
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
```

Comment:
(at least gpio and usb, i2c?)

```
supported:                                                                                                                                                                   
  - gpio                                                                                      
  - usb_device                                                                                

```

Reply:
I'll add gpio & usb - i2c depends on #14128

Reply:
Ah turns out this makes CI build some more projects with the board!
The build failure should be fixed by #14832 then. I'd rather have this merged first than manually enabling the driver implementations to the board default config.

Reply:
It still fails!
#15584 should fix this :crossed_fingers: